### DISCUSSION TITLE
Suggestions for future Administration features plan

### DISCUSSION THREAD

**Discussion by Krr0ptioN** [marzneshin/marzneshin#171] [Ideas]
> با عرض درود و خسته نباشید درخواست داشتم تا در صورت امکان این قابلیت ها رو به مرزنشین اضافه کنید :
> 
> * [x]  #135
> * [ ]  #136
> * [x]  #138
> * [ ]  در بخش داشبورد درصد استفاده از منابع مانند هسته سی پی یو و هارد دیسک اضافه شود
> * [x]  #137
> * [x]  فلو پیشفرض :
>   flow : xtls-rprx-viosion
>   رو ادمین روت بتونه تعین کنه و هر ادمینی خواست reality استفاده کنه این فلو به صورت دیفالت بزایش تعین بشه و غیرقابل تغییر باشه براش مگر درصورت صلاح دید ادمین روت
  > * [x]  لاگ هسته به صورت پیشفرض چند ثانیه اجرا می شود و سپس قطع می شود و سپس بعد ۱ دقیقه دوباره شروع میشود ، بعد نیست قابلیت استارت و استاپ دستی بهش اضافه بشود
> * [x]   #64 
>
> اینها به نظرم جاشون خالیه و اگر اضافه شوند بسیار عالی می شود با تشکر 🙏❤️

Reference: #128

**Comment by Krr0ptioN** [comment]
فیچیر های مربوط به مدیریت چند ادمینه رو از طریق #73 دنبال کن.

**Comment by w0l4i** [comment]
ممنون بابت پیگیری واقعا دارید عالی پیش میرید 

please add auto refresh button for users and static's (every x sec can be chosen by admin on this page)
also need this data's too :
on users segments show which user created by which admin , and also quick look admins can be awesome "sorting each admin users individually by admin's that created users on same page by sudo admin "

then other data's needed too :
when sorting by admin x for example created users on this page indeed it's needed to be added this feature that shows when user y was made and when was the latest edit and also resets , data limit , time limit changes to be shown
also one last thing is users have been deleted by admin x should be shown on some where

**Reply by Krr0ptioN** [reply]
> it's needed to be added this feature that shows when user y was made and when was the latest edit and also resets , data limit , time limit changes to be shown also one last thing is users have been deleted by admin x should be shown on some where.

- #172

**Reply by Krr0ptioN** [reply]
> on users segments show which user created by which admin , and also quick look admins can be awesome "sorting each admin users individually by admin's that created users on same page by sudo admin "

- #136
- #135
